pkg/apis/kops/v1alpha1/zz_generated.conversion.go

```diff
@@ -200,6 +201,9 @@ func Convert_kops_ClassicNetworkingSpec_To_v1alpha1_ClassicNetworkingSpec(in *ko
 }
 
 func autoConvert_v1alpha1_Cluster_To_kops_Cluster(in *Cluster, out *kops.Cluster, s conversion.Scope) error {
+	if err := api.Convert_v1_TypeMeta_To_v1_TypeMeta(&in.TypeMeta, &out.TypeMeta, s); err != nil {
```

Comment:
This code looks fine, and seems harmless - but the lazy engineer inside wants this to happen automagically somehow..

Maybe think about how we could implement a function call in the conversion code programmatically before calling the conversion function itself..? Just a thought